aten/src/ATen/THLongStorageView.h

```diff
@@ -13,11 +13,7 @@ static inline bool is_noelem_tensor_size(ArrayRef<int64_t> size) {
 }
 
 enum class THLongStorageViewKind {
-  SIZE,
```

Comment:
`is_noelem_tensor_size` (right above this line, GH won't let me coment on it directly) is dead code; get rid of ti.